common/supplemental/metaZones.xml

```diff
@@ -1390,9 +1390,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 				<usesMetazone from="1999-10-31 01:00" mzone="Europe_Eastern"/>
 			</timezone>
 			<timezone type="Europe/Volgograd">
-				<usesMetazone to="1992-03-28 22:00" mzone="Volgograd"/>
-				<usesMetazone to="2018-10-27 23:00" from="1992-03-28 22:00" mzone="Moscow"/>
```

Comment:
After reviewing 2020e data changes for Volgograd, I was wondering if we should add metazone mapping to Moscow at the end. It is true that Volgograd use same offset with Moscow, and 1 hour ahead back and forth. We already created a metazone dedicated for Volgograd. I also reviewed old tz database, but it assigned abbreviation separated from Moscow for this period. My conclusion is it makes more sense to use metazone Volgograd all the time.